src/ignitus-StudentLogin/Components/Login.js

```diff
@@ -1,45 +1,58 @@
-import React, { Component } from 'react';
-import _ from 'lodash';
-import { Redirect, Link } from 'react-router-dom';
-import PropTypes from 'prop-types';
-import { withErrorBoundary } from '../../ignitus-Internals';
+import React, { Component } from "react";
+import _ from "lodash";
+import { Redirect, Route, Link } from "react-router-dom";
+import { withErrorBoundary } from "../../ignitus-Internals";
 import * as t from './Constants';
-import loader from '../../ignitus-Assets/Images/loader2.gif';
-import '../Styles/style.scss';
+import loader from "../../ignitus-Assets/Images/loader2.gif";
+import "../Styles/style.scss";
 
 class Login extends Component {
   constructor(props) {
     super(props);
-    this.state = { email: '', password: '', emptymessage: false };
+    this.state = { email: "", password: "", emptymessage: false, invalidEmail: false  };
     this.handleSubmit = this.handleSubmit.bind(this);
+
   }
 
   handleSubmit(e) {
     e.preventDefault();
 
     const {
-      state: { email, password },
+      state: { email, password }
     } = this;
 
-    if (!_.isEmpty(email) && !_.isEmpty(password)) {
-      this.props.logInRequest(email, password);
-      this.setState({ email: '', password: '', emptymessage: false });
-    } else {
-      this.setState({ emptymessage: true });
+    if(_.isEmpty(email) || _.isEmpty(password)){
+      this.setState({emptymessage: true})
+      return
     }
+
+    if(typeof email !== "undefined"){
+      let lastAtPos = email.lastIndexOf('@');
+      let lastDotPos = email.lastIndexOf('.');
+
+      if (!(lastAtPos < lastDotPos && lastAtPos > 0 && email.indexOf('@@') == -1 && lastDotPos > 2 && (email.length - lastDotPos) > 2)) {
+        this.setState({invalidEmail: true})
+        return 
+      }
+    } 
+
+    this.props.logInRequest(email, password);
+    this.setState({ email: "", password: "", emptymessage: false });
+
   }
 
   render() {
+    console.log('msg', this.props.studentLoginData)
```

Comment:
Can you please remove the console statements from the commit

Reply:
Sure